app/views/users/show.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1> Hello views </h1>
-<p> this is a test</p>
\ No newline at end of file
+<div class="user">
+<%= render 'partials/user', user: @user %>
+<div class='bio'>
+<p><%= @user.bio %></p>
+</div>
+<ul class='card'>
+    <% @user.recent_three_posts.each do |post| %>
+      <a href="/users/<%= @user.id %>/posts/<%= post.id %> ">
+        <%= render 'partials/post', post: post %>
```

Comment:
[ OPTIONAL ] An error occurs when user clicks on `comments` or `likes` of any post. Kindly fix the error.

![posts_show](https://user-images.githubusercontent.com/35504149/205401577-8cab633d-a3ef-45ab-939a-57192554215b.png)